integration/kubernetes/overhead/steps/setup.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+RUNTIMECLASS=${RUNTIMECLASS:-kata}
```

Comment:
This can be defined at the `yamls` so it follows the structure of the current k8s yamls

Reply:
yamls can not definied variables that are modified at runtime, I added this to be flexible and anybody can run in an exiting  running cluster where the runtimeclass  name chage. For example kata deploy does not provide a runtimeclass named `kata`. I have open one issue related to it.